executor/executor.go

```diff
@@ -97,6 +97,7 @@ const (
 // Otherwise the executor's returned rows don't need to store the handle information.
 type Row = types.DatumRow
 
+// baseExecutor contains an Executor's array and it implements visitor pattern.
```

Comment:
// baseExecutor contains the common resource needed by every "Executor", including children "Executor"s, execution context, schema.

Reply:
It has not implemented the visitor pattern.

Reply:
Per my understanding, that is visitor pattern.

Reply:
But it's different from the traditional, common used visitor pattern, witch may further cause some misunderstanding.